core/src/test/java/org/apache/accumulo/core/file/blockfile/cache/TestLruBlockCache.java

```diff
@@ -76,7 +76,9 @@ public void testConfiguration() {
     // this is not set for the cache type, so should fall back to default
     cc.set(defaultPrefix + LruBlockCacheConfiguration.MAP_LOAD_PROPERTY, "0.53");
 
-    BlockCacheConfiguration bcc = new BlockCacheConfiguration(cc);
+    BlockCacheConfiguration bcc = new BlockCacheConfiguration(cc, Property.TSERV_PREFIX,
+        Property.TSERV_INDEXCACHE_SIZE, Property.TSERV_DATACACHE_SIZE,
+        Property.TSERV_SUMMARYCACHE_SIZE, Property.TSERV_DEFAULT_BLOCKSIZE);
```

Comment:
This pattern keeps getting repeated. It's pretty verbose boilerplate to construct the block cache configuration now. It'd probably be easier to pass a flag/enum to specify whether TSERV or SSERV props should be used, and then resolve that logic inside the constructor.

Reply:
Addressed in 9ec7f5f. I modified BlockCacheConfiguration to create two static methods that returned either a configuration for the TabletServer or the ScanServer.